src/openrct2-ui/windows/Ride.cpp

```diff
@@ -5099,13 +5081,15 @@ static void window_ride_music_mousedown(rct_window* w, rct_widgetindex widgetInd
             }
         }
     }
-
-    // Sort available music by the original RCT2 list order
-    std::stable_sort(musicOrder.begin(), musicOrder.end(), [](const ObjectEntryIndex& a, const ObjectEntryIndex& b) {
-        auto orderA = GetMusicStyleOrder(a);
-        auto orderB = GetMusicStyleOrder(b);
-        return orderA < orderB;
-    });
+    std::string bufferA;
```

Comment:
I'd just move these into the lambda. Yeah it might allocate a bit more but it's not the end of the world